$MirOS: ports/mailnews/pine/patches/patch-pine_osdep_os-bso_h,v 1.2 2005/05/04 23:37:41 tg Exp $

ATTN: first chunk must read @ETC@/pine.conf*

--- pine/osdep/os-bso.h.orig	Fri May 23 16:04:59 2003
+++ pine/osdep/os-bso.h	Thu Dec 15 13:48:04 2005
@@ -163,10 +163,10 @@
 
 /*----- System-wide config file ----------------------------------------*/
 #ifndef SYSTEM_PINERC
-#define SYSTEM_PINERC             "/etc/pine.conf"
+#define SYSTEM_PINERC             "@ETC@/pine.conf"
 #endif
 #ifndef SYSTEM_PINERC_FIXED
-#define SYSTEM_PINERC_FIXED       "/etc/pine.conf.fixed"
+#define SYSTEM_PINERC_FIXED       "@ETC@/pine.conf.fixed"
 #endif
 
 
@@ -256,7 +256,7 @@
  in the user's home directory and have a number appended to them when
  there is more than one.
  ----*/
-#define DEBUGFILE	".pine-debug"
+#define DEBUGFILE	".etc/pine-debug"
 
 /*----------------------------------------------------------------------
     The number of debug files to save in the user's home diretory. The files
@@ -295,7 +295,10 @@
 #define MAX_ADDR_EXPN   (1000)  /* Longest expanded addr                    */
 #define MAX_ADDR_FIELD (10000)  /* Longest fully-expanded addr field        */
 
+/*
+ * File name used to store the user's server/id/password triple between
+ * session.  It is rooted in the same directory as the PINERC.
+ */
+#define	PASSFILE	"ssh/pine.pwd"
 
 #endif /* _OS_INCLUDED */
-
-
